Resources/Prototypes/Research/experimental.yml

```
@@ -157,3 +157,15 @@
   recipeUnlocks:
     - WeaponForceGun
     - WeaponTetherGun
+
+- type: technology
+  id: SpeedTech
+  name: research-technology-speed-tech
+  icon:
+    sprite: Clothing/Shoes/Boots/speedboots.rsi
+    state: icon
+  discipline: Experimental
+  tier: 3
+  cost: 15000
```

```suggestion
  cost: 7500
```
really steep for one item